### DISCUSSION TITLE
课程整体安排的讨论

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/OSSDevGov2021#3] [01 课程资源发布]
## 学习目标与要求

* 了解全球的大规模软件协作开发生产模式与开源软件背后的缘起；
* 了解开源软件工程，掌握开源软件工程的工程方法，能够通过Git协作工具完成开源软件工程的整个流程；
* 了解开源项目的开发模式，能够以贡献者的角色参与到实际的开源项目开发中贡献代码；
* 了解开源项目的管理机制，能够以管理者角色参与到实际的开源项目开发管理中；
* 了解开源社区的运作模式，能够以社区运营者角色参与到实际开源社区的治理中；
* 了解开源社区的运营的艺术，能够通过数字化工具，衡量开源社区的健康状况，进而制定各种策略，保障开源社区的健康发展。

## 课程授课信息

* 时间：每周一下午，第7 - 8节课（3点 - 4点40）
* 地点：华东师大，中北校区，计算机楼115教室

## 课程内容与安排

课程分为两个部分：

- 5月1日之前，重点是：**Part 01：开源软件开发**
- 5月1日之后，重点是：**Part 02：开源社区治理**

大家对课程有什么想法，可以在这里提出

**Comment by frank-zsy** [comment]
Readme 中的 「The Code」和「开源 60 年」都非常有价值，是否可以考虑将 [Revolution OS](https://www.bilibili.com/video/BV1as411f7nM) 也作为参考资料？

**Reply by will-ww** [reply]
嗯，已加，前面是石墨加过，没有更新过来~
